irods/connection.py

```diff
@@ -437,13 +456,32 @@ def _login_gsi(self):
 
     def _login_pam(self):
 
-        time_to_live_in_seconds = 60
-
-        pam_password = PAM_PW_ESC_PATTERN.sub(lambda m: '\\'+m.group(1), self.account.password)
+        import irods.client_configuration as cfg
+        inline_password = (self.account.authentication_scheme == self.account._original_authentication_scheme)
+        # By default, let server determine the TTL.
+        time_to_live_in_hours = 0
+        new_pam_password = PAM_PW_ESC_PATTERN.sub(lambda m: '\\'+m.group(1), self.account.password)
```

Comment:
please add a comment for what this line is doing and why

Reply:
Line 463 was your meaning, correct?

Reply:
Comment added